### PR TITLE
Update compliance-checker to 2.0.0 ...

### PR DESCRIPTION
**Pull request by ocefpaf**
... and added the glider plugin.

### CODE REVIEW

**Comment by ocefpaf** [comment]
ping @lukecampbell.

People can `conda install -c ioos compliance-checker cc-plugin-glider` as soon as I merge this.

**Comment by ocefpaf** [comment]
The AppVeyor is due to an update in conda.  I am merging this to the the Linux and OSX packages up.

**Comment by abirger** [comment]
@ocefpaf , in layman words - what does that AppVeyor issue mean for conda in Windows? Should we wait for conda update?

**Comment by ocefpaf** [comment]
The problem is probably somewhere between a conda update and the build tool we use (obvious-ci). 

Conda just enabled a *Visual Studio Build String* feature (that `vc14` (or `vc8`, `vc10`, etc) you see when you install a package on Windows) and apparently all the packages must have that now or we will get a bogus conflict like the one below:

```
Hint: the following packages conflict with each other:
  - obvious-ci
  - python 3.5*
 
Use 'conda info obvious-ci' etc. to see the dependencies for each package.
 
Note that the following features are enabled:
  - vc14
```

What is even more puzzling is the fact that all python versions are falling back to Python 3.5 at build time!!!  I am debugging this...  Hang on.

**Comment by abirger** [comment]
@ocefpaf , wait, wait... should it be Python 3? My conda installed Python 2.7.10.

**Comment by ocefpaf** [comment]
It should be `Python 2.7`, the fact that it is ignoring that and trying to build 3.5 is part of the problem.

**Comment by abirger** [comment]
OK, got it...
